packages/components/pie-card/src/index.ts

```diff
@@ -53,28 +53,36 @@ export class PieCard extends LitElement implements CardProps {
     @validPropertyValues(componentSelector, paddingValues, undefined)
     public padding?: CardProps['padding'];
 
+    private onClickHandler (event: Event) {
+        if (this.disabled) {
+            event.preventDefault();
+            event.stopPropagation();
+        }
+    }
```

Comment:
Is this handler really necessary?

Reply:
yes, we need to disable the component's click event if `disabled` is passed.
Since we use a div with the role of a button for the card, we have two options to disable the component: intercept the click event or use `pointer-events: none`. I opted in to intercept the event because disabling interactivity with CSS (pointer-events: none) doesn't prevent events triggered by JavaScript/programmability. For example, if you query the element,  and call it using `element,clikc()`.

Reply:
Good one! Would a short comment make sense before `event.preventDefault ` and `event.stopPropagation` so it becomes clear for others and they will not possibly risk removing these lines?

Reply:
done 💯